src/components/scenes/StakingChangeScene.js

```diff
@@ -205,15 +205,18 @@ export const StakingChangeSceneComponent = (props: Props) => {
   }, [amount, currencyPlugin])
 
   useEffect(() => {
-    if (!selectedFioAddress) {
-      const fioAddress = fioAddresses.find(({ walletId: fioAddressWalletId }) => fioAddressWalletId === walletId)
+    if (!selectedFioAddress && fioAddresses != null && fioAddresses.length > 0) {
+      const fioAddress = fioAddresses
+        .filter(({ walletId: fioAddressWalletId }) => fioAddressWalletId === walletId)
+        .sort(({ bundledTxs }, { bundledTxs: bundledTxs2 }) => (bundledTxs <= bundledTxs2 ? 1 : -1))[0]
 
       // If no address is found, we do not define the selectedFioAddress
       if (fioAddress == null) return
+      if (fioAddress.bundledTxs < 2) return setError(new Error(s.strings.staking_no_bundled_txs_error))
```

Comment:
Add a comment explaining the purpose of this line.

Reply:
```js
// Restrict staking to addresses with only 2 or more bundled transactions to avoid user needing to pay fee
```

Reply:
Double-check what the minimum bundleTx value is (1 or 2)

Reply:
Move to accountbased instead of in the GUI (if it's not too difficult).

Reply:
We're not going to worry about moving the `bundledTxs` logic to the plugin for now.